internal/service/ldapverify/data_source_ldap_verify_test.go

```diff
@@ -12,21 +12,19 @@ import (
 func TestAccLDAPVerifyDS_basic(t *testing.T) {
 	var (
 		dataSourceName = "data.mongodbatlas_ldap_verify.test"
-		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
 		hostname       = os.Getenv("MONGODB_ATLAS_LDAP_HOSTNAME")
 		username       = os.Getenv("MONGODB_ATLAS_LDAP_USERNAME")
 		password       = os.Getenv("MONGODB_ATLAS_LDAP_PASSWORD")
 		port           = os.Getenv("MONGODB_ATLAS_LDAP_PORT")
-		projectName    = acc.RandomProjectName()
-		clusterName    = acc.RandomClusterName()
+		projectID, _   = acc.ClusterNameExecution(t)
 	)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
tests can run in parallel as each ldap-verify resource has its unique request id, no conflicting in running at the same time using the same project and cluster.